src/Horse.Core.Param.pas

```diff
@@ -147,6 +148,12 @@ function THorseCoreParam.Field(const AKey: string): THorseCoreParamField;
   end;
 end;
 
+function THorseCoreParam.AddOrSetValue(const AKey: string; const AValue: string): THorseCoreParam;
```

Comment:
These parameters come from the Request, that is, they shouldn't be changers and added values ​​on the outside.